DNN Platform/HttpModules/Membership/MembershipModule.cs

```diff
@@ -256,5 +257,38 @@ public static void AuthenticateRequest(HttpContextBase context, bool allowUnknow
                 context.Items.Add("UserInfo", new UserInfo());
             }
         }
+
+        private static bool NeedLogout(HttpContextBase context, UserInfo user)
```

Comment:
Recommend changing this to RequiresLogout

Reply:
Updated.